types/trace/network_trace.go

```diff
@@ -189,3 +193,31 @@ type ProtoDNSOPT struct {
 	Code string `json:"code"`
 	Data string `json:"data"`
 }
+
+//type ProtoHTTP struct {
```

Comment:
I'll wait for the PR adding the events to be ready first, but I left you a question there asking if we shouldn't have a single HTTP event (like the DNS one) and differentiate request/response in the derivation layer (like the DNS as well). We can discuss this in that PR (and then revisit this one for the review).